resumeform.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <form>
+            <label for="fname"> First Name</label>
+            <input type="text" id="fname" name="fname" maxlength="30"><br>
+
+            <label for="lastname">Last name:</label>
+            <input type="text" id="lname" name="lname" maxlength="30"><br>
+
+            <label for="contactphonenumber"> Contact Phone Number></label>
+            <input type="text" id="contactphonenumber" name="contactphonenumber" maxlength="50"><br>
+            
+
```

Comment:
Missing (-40)
- Phone number type: Landline, Mobile (radio button)
- Contact email (text field, limit of 50 characters)
- How did you hear about us: Search Engine, Social Media, Recruiter (drop down)
- Interest for contacting: A list of your skills (check box)
- Message (text area, limit of 250 characters)
- Use of labels with all form fields
- Use of validation

57/100
Finish the lab and your grade will improve